src/ProvidedTypes.fsi

```diff
@@ -330,6 +330,9 @@ namespace ProviderImplementation.ProvidedTypes
         /// The documentation is re-computed  every time it is required.
         member AddXmlDocComputed: xmlDocFunction: (unit -> string) -> unit
 
+        /// Add the attributes on the provided type.
+        member AddAttributes: TypeAttributes -> unit
```

Comment:
Added this member to `ProvidedTypeDefinition` in order to more easily add attributes for abstract classes. May want to move this to another ctor parameter.